### PR TITLE
DevTools: Rendering feature reference

### LINKED ISSUE #2543: DevTools v99: Rendering > Auto dark mode
Document https://developer.chrome.com/blog/new-in-devtools-99/#auto-dark-mode

### LINKED ISSUE #2544: DevTools: Rendering reference

### PR DESCRIPTION
**Pull request by sofiayem**
Fixes #2544 
Fixes #2543 

Wanted to document the [Auto Dark theme](https://developer.chrome.com/blog/new-in-devtools-99/#auto-dark-mode) but the entire Rendering tab was undocumented.

So, documented the Rendering tab.

### CODE REVIEW

File: site/en/docs/devtools/rendering/index.md [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+To disable local fonts:
+
+1. [Open the **Rendering** tab](#open-rendering) and check **Disable local fonts**.
+1. <span class="material-icons">refresh</span> Reload the page and observe the font of the sentence above change to Roboto.
+
+## Emulate a focused page
+
+If you switch focus from the current page to DevTools, some overlay elements automatically hide if they are triggered by focus. For example, dropdown lists, menus, or date pickers.
+
+To emulate a focused page:
```
Comment (jecfish) [inline site/en/docs/devtools/rendering/index.md:194]: Can improve on the example as the effect is not obvious.A dropdown example is better or a demo video. 

Refer to demo at 2:00. https://portal.gitnation.org/contents/debugging-with-chrome-devtools

File: site/en/docs/devtools/rendering/index.md [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/peTdHnJJeksHISUdpJzX.png", alt="Emulated prefers-color-scheme:dark", width="800", height="487" %}
+
+## Emulate CSS media type (Enable print preview)
+
+The [print media query][1] controls how your page looks when printed.
+
+To force your page into print preview mode:
+
+1.  Open the **Rendering** tab](#open-rendering) and under **Emulate CSS media type** select **print**.
```
Comment (jecfish) [inline site/en/docs/devtools/rendering/index.md:242]: broken link

File: site/en/docs/devtools/rendering/index.md [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+   - `prefers-contrast:less`
+   - `prefers-contrast:custom`
+
+## Emulate CSS media feature `prefers-reduced-motion`
+
+The [`prefers-reduced-motion` CSS media feature][8] indicates if the user has requested to minimize the amount of motion on a page.
+
+To emulate this condition:
+
+1. [Open the **Rendering** tab](#open-rendering) on the [demo page](https://prefers-reduced-motion.glitch.me/) and try scrolling to see various animations.
```
Comment (jecfish) [inline site/en/docs/devtools/rendering/index.md:285]: Probably can create an easier demo here right in the page:
- e,g. https://jec.fyi/demo/prefers-reduced-motion, or
- a box with hover effect (e.g. https://jec.fyi/demo/css-grid-cq-cascade-2): -
    - no preference: transform bigger
    - reduced: only highlight the border

File: site/en/docs/devtools/rendering/index.md [hunk 4]
```diff
@@ -0,0 +1,356 @@
...
+## Emulate vision deficiencies
+
+Everyone should be able to access and enjoy the web. [Google is committed to making that a reality](https://www.google.com/accessibility/).
+
+With Chrome DevTools, you can see how people with vision deficiencies see your site, so you can make it better for them.
+
+To emulate vision deficiencies:
+
+1. [Open the **Rendering** tab](#open-rendering).
+1. Under the **Emulate CSS media feature `color-gamut`**, select one of the following from the dropdown list:
```
Comment (jecfish) [inline site/en/docs/devtools/rendering/index.md:312]: incorrect media feature

File: site/en/docs/devtools/rendering/index.md [hunk 5]
```diff
@@ -0,0 +1,356 @@
...
+
+   - No emulation
+   - Blurred vision
+   - Protanopia—can't perceive red and green colors
```
Comment (jecfish) [inline site/en/docs/devtools/rendering/index.md:316]: Inconsistent options styles. Do we not bold / bold the options or use back tick?

File: site/en/docs/devtools/rendering/index.md [hunk 6]
```diff
@@ -0,0 +1,356 @@
...
+   - Deuteranopia—confusion of red, green, and yellow
+   - Tritanopia—can't distinguish blue and yellow
+   - Achromatopsia—partial or total absence of color vision
+
+  For example:
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/aj8UMMDpofn8s7UPzHTy.png", alt="Emulated blurred vision", width="800", height="481" %}
+
```
Comment (jecfish) [inline site/en/docs/devtools/rendering/index.md:324]: Include resource to read further: https://developer.chrome.com/blog/cvd/

File: site/en/docs/devtools/rendering/index.md [hunk 7]
```diff
@@ -0,0 +1,356 @@
...
+## Disable AVIF image format
```
Comment (jecfish) [inline site/en/docs/devtools/rendering/index.md:325]: provide example and use case, e.g. https://twitter.com/jecfish/status/1504658919003615240

**Review by jecfish — CHANGES_REQUESTED** [review]

**Comment by sofiayem** [comment]
> Performance related emulation
> CSS media features emulation
> Others

So, I assume the the "Emulate a focused page" should go into "Others" then. Then the reader is gonna have to guess that the focused-related checkbox is in Rendering > Others in table of contents. It's not easy to do so.

But let me think about it more.

File: site/en/docs/devtools/css/print-preview/index.md [hunk 8]
```diff
@@ -1,38 +0,0 @@
----
-layout: "layouts/doc-post.njk"
```
[the remark this answers is not in the document]
Comment (sofiayem) [reply site/en/docs/devtools/css/print-preview/index.md:2]: Riiight. Forgot about it. Thank you for catching this.

File: site/en/docs/devtools/rendering/apply-effects/index.md [hunk 9]
```diff
@@ -0,0 +1,142 @@
...
+<link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
+
+Discover useful effects to apply to your page with this reference of the **Rendering** tab options.
+
+## Highlight ad frames
+
+To check if frames are tagged as ads:
+
+1. [Open the **Rendering** tab](/docs/devtools/rendering#open-rendering)  on this [demo ](https://heavy-ads.glitch.me/) and check **Highlight ad frames**.
```
Comment (jecfish) [inline site/en/docs/devtools/rendering/apply-effects/index.md:24]:
```suggestion
1. [Open the **Rendering** tab](/docs/devtools/rendering#open-rendering) on this [demo](https://heavy-ads.glitch.me/) and check **Highlight ad frames**.
```

**Review by jecfish — APPROVED** [review]

**Pull request merged**